### PR TITLE
Sets the Operator Image Pull Policy to IfNotPresent

### PR DESCRIPTION
**Pull request by danehans**
https://github.com/projectcontour/contour-operator/pull/162 set `imagePullPolicy: Always` for the operator. This PR updates the value to `IfNotPresent` (default) to be consistent with Contour [examples](https://github.com/projectcontour/contour/tree/main/examples/contour).

Signed-off-by: Daneyon Hansen <daneyonhansen@gmail.com>

### CODE REVIEW

**Review by jpeach — APPROVED** [review]

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
         args:
         - --enable-leader-election
         image: contour-operator
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```
Comment (stevesloka) [inline config/manager/manager.yaml:28]: Should this be left as `Always` in a dev build scenario where you'd push the same image tag over and over testing?

Comment (skriss) [reply config/manager/manager.yaml:28]: In contour we leave the "source" YAML files as `Always` (for dev), then update it to `IfNotPresent` for the rendered file, which is the one that gets served for the quickstart link. It'd be nice to do the same here, I think.

**Comment by danehans** [comment]
@stevesloka @skriss commit `12dcde1` resolves https://github.com/projectcontour/contour-operator/pull/166#discussion_r545193187. The pull policy will continue to be set as `Always`. When a release is cut, i.e. `hack/release/make-release-tag.sh`, the policy will be updated to `IfNotPresent`. PTAL when you have a moment.

**Review by stevesloka — APPROVED** [review]
/lgtm

**Review by skriss — APPROVED** [review]
LGTM :+1:

**Pull request merged**